docs/source/tutorials/multifidelity/README.rst

```diff
@@ -18,3 +18,4 @@ problems.
    mf_sync_model
    mf_async_model
    mf_comparison
+   mf_pasha
```

Comment:
Please remove this. The MF tutorial is not for special extensions like PASHA. You can link the PASHA tutorial if you like.

Reply:
I've removed this now